index.html

```diff
@@ -29,8 +30,22 @@
 			<h1> Jamila Ferron</h1>
 			<p> Full-stack web developer with an interest in PHP web development</p>
 		</div>
-		<img class="scroll-icon" src="img/download.svg" alt=""/>
+		<a href="#contact-section"><img class="scroll-icon" src="img/download.svg" alt=""/></a>
 	</header>
+	<section class="container" id="contact-section">
+		<div class="contact-wrapper">
+			<div class="section-header">
+				<h1 class="front-text">
+					Contact
+				</h1>
+				<h1 class="shadow-text">
+					Reach Out!
+				</h1>
+			</div>
+			<a href="mailto:jamila_ferron@hotmail.co.uk"><img class="contact-icon" src="img/envelope.svg" alt=""/></a>
```

Comment:
Not sure why but the clickable area is quite tall on the envelope. Would be better if it was limited to the icon itself.